pkgs/fish-shell/fish-shell/registry.yaml

```diff
@@ -4,13 +4,10 @@ packages:
     repo_owner: fish-shell
     repo_name: fish-shell
     description: The user-friendly command line shell
-    asset: fish-static-{{.OS}}-{{.Version}}.{{.Format}}
+    asset: fish-static-{{.Arch}}-{{.Version}}.{{.Format}}
     format: tar.xz
     replacements:
       arm64: aarch64
-      linux: amd64
-    supported_envs:
-      - linux
```

Comment:
Why did you remove `supported_envs`?
I think the static binary supports only Linux.